server/src/http/server.ts

```diff
@@ -482,66 +479,23 @@ function setupRoutes(app: Application) {
         requireOrganismePermission("manageEffectifs"),
         express
           .Router()
-          .get("/", validateRequestMiddleware({ body: uploadedDocumentSchema() }), async (req, res) => {
-            return upload.getDocument(res.locals.organismeId, req.body, res);
-          })
-          .post("/", async (req, res) => {
-            return upload.createUpload(res.locals.organismeId, req, res);
-          })
-          .get(
-            "/get",
-            returnResult(async (req, res) => {
-              return upload.getUpload(res.locals.organismeId);
-            })
-          )
-          .post(
-            "/pre-import",
-            validateRequestMiddleware({ body: uploadMappingSchema() }),
-            returnResult(async (req, res) => {
-              let userMapping = req.body;
-              return upload.preImport(res.locals.organismeId, req.user, userMapping);
-            })
-          )
           .post(
-            "/import",
-            returnResult(async (req, res) => {
-              return upload.import(res.locals.organismeId, req.user);
+            "/validate",
+            returnResult(async (req) => {
+              const data = await z.array(dossierApprenantSchemaV3()).safeParseAsync(req.body);
+              return data;
```

Comment:
L'objetif de cette route est de valider que le fichier a le bon format ? Est-ce qu'on ne pourrait pas faire les mêmes contrôles que dans l'ingestion (maintenant que c'est plutôt rapide) : vérifier si la formation existe, idem pour l'organisme. Je ne sais pas si ça a une utilité.
Je me dis qu'il faudrait savoir quel organisme peut importer quoi...

Reply:
Oui en itération c'est beaucoup mieux !!!! <3